pyiron_continuum/damask/rolling.py

```diff
@@ -0,0 +1,363 @@
+from ast import arg
+import imp
+import os
+from random import vonmisesvariate
+import numpy as np
+import matplotlib.pyplot as plt
+
+import sys
+import os
+import subprocess
+
+dir = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, dir)
+
+from damaskjob import DAMASK
+from damask import regridding as rgg
```

Comment:
@yangbai90 `regridding` cannot be found anywhere in `pyiron_continuum` or on [DAMASK GitHup page](https://github.com/eisenforschung/DAMASK/tree/master/python/damask).